assets/css/style.css

```diff
@@ -198,25 +198,30 @@ a:hover {
 
 .article-btn {
   padding: 12px;
-  border-radius: 12px;
```

Comment:
- [optional] Please add the active state on the mobile as it will be used to identify an active button on the webpage, 

```css
.article-btn:active {
background-color: #5280f5;
  color: #fff;
  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
}
```